### DISCUSSION TITLE
Any reason why we are raising an exception here?

### DISCUSSION THREAD

**Discussion by thanhlim** [logandk/serverless-wsgi#227] [General]
https://github.com/logandk/serverless-wsgi/blob/72de52524a2fc393120aa11d64df946c9aa358a5/serverless_wsgi.py#L353

``` 
if response.status_code >= 300:
        raise RuntimeError(json.dumps(returndict))
```

This doesn't seem to respect the HTTP response here if we have Flask behind it.  A 400 error by the service then is raised and this RunTimeError gets raised.  What is the intent of that and what's a better way of returning the error, where we know that the client has parameters that are invalid as they call into our wsgi service?

**Comment by logandk** [comment]
Not sure. @jvarho did the lambda integration, maybe they can share some insight. If not, I'm fine with removing this.

**Comment by jvarho** [comment]
See #167 for the details, but basically you need to add a status code mapping pattern for any errors you want to support. The alternative would have been setting http status automatically, but not allowing custom mappings.

Since one reason you'd use lambda integration is for control over mappings, I figured the clunky option was better. But I still only use it for async so I don't mind.
